applications/openshift/ocp_version_yaml_path.var

```diff
@@ -0,0 +1,21 @@
+documentation_complete: true
+
+title: 'OCP version yaml path'
+
+description: |-
+    When scanning OpenShift clusters, not all type of cluster has same api
+    resource path for ocp version, ex. HyperShift OCP version api resource path
+    is different than a regular OpenShift Cluster.
+    This variable determines what yaml api filter to apply to the fetched api resource
+    defined by ocp_version_api_path variable.
+    The default value is for a regular OpenShift Cluster. this variable should be used
+    in conjunction with ocp_version_api_path.
+
+type: string
+
+operator: equals
+
+interactive: false
+
+options:
+    default: "[.status.versions[].version]"
```

Comment:
Why do we have the square braces around the path? Is it because `[shared/checks/oval/installed_app_is_ocp4.xml](https://github.com/ComplianceAsCode/content/pull/9726/commits/563a00367342be6f7d5ed17d2c27add3d0a1cc89#diff-79f1a46adfc289cd2d6b69c96af6c3a98e38ba749db957fcb7c823ac1a4430f6)` uses a list expansion so we need to force a list here?

Reply:
Actually, I am not sure if we need the square braces, let me try without it